packages/cli/test/code-pushup.config.ts

```diff
@@ -19,29 +28,26 @@ export default {
           slug: 'dummy-audit-1',
           title: 'Dummy Audit 1',
           description: 'A dummy audit to fill the void 1',
-          label: '???',
           docsUrl: 'http://www.my-docs.dev?slug=dummy-audit-1',
         },
         {
           slug: 'dummy-audit-2',
           title: 'Dummy Audit 2',
           description: 'A dummy audit to fill the void 2',
-          label: '???',
           docsUrl: 'http://www.my-docs.dev?slug=dummy-audit-2',
         },
         {
           slug: 'dummy-audit-3',
           title: 'Dummy Audit 3',
           description: 'A dummy audit to fill the void 3',
-          label: '???',
           docsUrl: 'http://www.my-docs.dev?slug=dummy-audit-3',
         },
       ],
       runner: {
         command: 'node',
         args: [
-          '-e',
-          `require('fs').writeFileSync('${outputDir}/dummy-plugin-output.json', '${JSON.stringify(
+          'echo',
```

Comment:
I guess `echo` must be like a command, not an argument. The example here #131

Reply:
Yes, will change it!